lib/smart_proxy_dynflow_core/settings.rb

```diff
@@ -40,6 +40,7 @@ class Settings < OpenStruct
         :plugins => {},
         :pid_file => '/var/run/foreman-proxy/smart_proxy_dynflow_core.pid',
         :daemonize => false,
+        :execution_plan_cleaner_age => 60 * 60 * 24,
```

Comment:
You need to add this also to `PLUGIN_SETTINGS` for the code to properly load the data from settings